.github/workflows/release-please.yml

```diff
@@ -51,17 +51,19 @@ jobs:
           # use a token so that workflows on the PR run when we push later
           token: ${{ steps.otelbot-token.outputs.token }}
 
-      # release-please does not do this on its own, so we do it here instead
+      # previous commiter does not have CLA signed, and the commit is missing updates to package-lock.json,
+      # so we re-write that commit here with the proper changes
       - name: Update package-lock.json in PR
         # only update if a PR has been created
         if: ${{ steps.release.outputs.pr }}
         run: |
+          git reset HEAD~1
           npm install --ignore-scripts --package-lock-only
-          git add package-lock.json
+          git add .
           git config user.name otelbot
           git config user.email 197425009+otelbot@users.noreply.github.com
-          git commit -m "chore: sync package-lock.json"
-          git push
+          git commit -m "chore: release main"
```

Comment:
Maybe use `git commit --amend` to avoid `git reset`?

Reply:
yep, that's better - thanks 🙂  applied in https://github.com/open-telemetry/opentelemetry-js-contrib/pull/3081/commits/a760bda2b590664b9e1dc103c6465584c5244207